iarc_arbiter/msg/RegisterBehavior.msg

```diff
@@ -1 +1,2 @@
-string name
\ No newline at end of file
+string name
+bool fast
```

Comment:
Consider better name for "fast" to indicate what it actually means.
Apparently, it decides between pass-through and looping modes.